spec/requests/physical_servers_spec.rb

```diff
@@ -304,4 +304,66 @@
       end
     end
   end
+
+  describe "Physical Server refresh action" do
+    context "with an invalid id" do
+      it "fails to refresh a physical server" do
```

Comment:
Nit, but `it "fails..."` suggests that something is wrong....possibly "it responds with 404 Not Found" or similar?